webapp/src/components/create_issue_menu/create_issue.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import GitLabIcon from '../../images/icons/gitlab';
+
+interface PropTypes{
+    show: boolean;
+    actions: any;
```

Comment:
Remove `any` and use proper type. Take reference from here - https://github.com/mattermost/mattermost-webapp/blob/4ed3c9728c0956c064a88b32c48edf3e24cc4bc0/components/profile_popover/profile_popover.tsx#L140